test/test-msbuild.sh

```diff
@@ -18,8 +18,13 @@
 
 for config in Debug Release; do
     for useenv in true false; do
-        EXEC "" ${BIN}msbuild /p:UseEnv=${useenv} /p:Configuration=${config} "${TESTS}HelloWorld.vcxproj"
-        rm -rf "${TESTS}/${config}"
+        # Trailing slash is required in MSBuild directory properties.
```

Comment:
Nit: there’s a typo in the commit message, “ atrifacts”. LGTM otherwise.

Reply:
Amended in https://github.com/mstorsjo/msvc-wine/pull/103/commits/1da913e9373f44a717221e6c972a98a737ab5b3d. Thanks!